effect/src/main/scala/scalaz/effect/ST.scala

```diff
@@ -134,8 +134,7 @@ object ST extends STInstances {
     private[effect] def run = f()
   }
 
-  // Implicit conversions between IO and ST
-  implicit def STToIO[A](st: ST[IvoryTower, A]): IO[A] =
+  def STToIO[A](st: ST[IvoryTower, A]): IO[A] =
```

Comment:
not specifically related to this PR, but would this be better defined as a `NaturalTransformation[ST[IvoryTower, _], IO]`?

Reply:
No preference